app/util/env.ts

```diff
@@ -21,11 +21,13 @@ try {
 }
 
 let envOverrides = {};
-try {
-  envOverrides = dotenv.parse(fs.readFileSync('.env'));
-} catch (e) {
-  winston.warn('Could not parse environment overrides from .env file');
-  winston.warn(e.message);
+if (process.env.NODE_ENV !== 'test') {
```

Comment:
This is unrelated to the current ticket, but should improve our tests by making sure that any custom .env values do not interfere.